src/components/Security/Users/EditCustomMapping.vue

```diff
@@ -126,6 +162,26 @@ export default {
     this.loading = false
   },
   methods: {
+    loadMappingValue(event) {
+      var file = event.target.files[0]
+      var reader = new FileReader()
+      reader.onload = async e => {
+        this.mappingValue = e.target.result
+        await this.onSubmit()
+      }
+      reader.readAsText(file)
+    },
+    downloadMappingValue() {
```

Comment:
I think create a link and trigger a click on it is not the best practice here.
You can find an example with better way to do this here: https://github.com/kuzzleio/kuzzle-admin-console/blob/4-dev/src/components/Common/Environments/EnvironmentsSwitch.vue#L94